ruoyi-common/src/test/java/com/ruoyi/common/support/ConvertTest.java

```diff
@@ -0,0 +1,205 @@
+package com.ruoyi.common.support;
```

Comment:
Does the PR title refer to the right `Convert` class?

Reply:
I missed that....  :O 
thanks